packages/atomic/src/components/common/query-summary/query-summary-common.tsx

```diff
@@ -88,23 +88,19 @@ export const QuerySummaryCommon: FunctionalComponent<
             params={paramsWithHighlights}
             count={params.count}
           />
-          <span
-            part="duration"
-            class={
-              props.enableDuration && props.querySummaryState.hasDuration
-                ? ''
-                : 'hidden'
-            }
-          >
-            <LocalizedString
-              key="in-seconds"
-              bindings={props.bindings}
-              params={{
-                count:
-                  props.querySummaryState.durationInSeconds.toLocaleString(),
-              }}
-            />
-          </span>
+          {props.withDuration && props.querySummaryState.hasDuration && (
+            <span part="duration">
```

Comment:
```suggestion
            <span class="hidden" part="duration">
```

Reply:
I'm unsure whether hiding the duration using a style should be something common shared across all components who may use QuerySummaryCommon, or if it's something that's only specific to `atomic-query-summary` 🤔 

Currently, this resides in `atomic-query-summary.pcss`.

Reply:
I think it's safe to say all query summary should have this hidden by default, if you think of Johnny's design as source of truth